extensions/DynamicPageList/DPLSetup.php

```diff
@@ -1409,7 +1409,7 @@ public static function dplNumParserFunction(&$parser, $text='') {
 		return $num;
     } 
 
-    public static function dplVarParserFunction(&$parser, $cmd) {
+    public static function dplVarParserFunction($parser, $cmd) {
 		$args = func_get_args();
         if      ($cmd=='set')     return DPLVariables::setVar($args);
         else if ($cmd=='default') return DPLVariables::setVarDefault($args);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)